e2e/condition_catalog_service_test.go

```diff
@@ -369,10 +369,6 @@ func TestCondition_CatalogServices_SharedDependency_NoServices(t *testing.T) {
 		name    string
 		runMode string
 	}{
-		{
```

Comment:
Removed this invalid test case. Empty string for `run` is not a valid value.

Reply:
actually in this case, we should verify that the nil case works. This test is verifying that the task creates without issue, and creating a task without a `run` parameter is a valid scenario from the API

Reply:
Good catch.
Added back a test case for creating a task without run mode: https://github.com/hashicorp/consul-terraform-sync/pull/826/commits/e8737b9bfd9fe4ec549f9e9dbe9b91a8a21d744d